### PR TITLE
adds check for token expiration

### PR DESCRIPTION
**Pull request by SwagOtaku**
fixes issue where users get errors out of the blue (due to Watchlist Flavor token expiring) by notifying user to relogin after 30 days.

set to 30 days since both WonderfulSubsWLF and KitsuWLF tokens expire in 30 days

### CODE REVIEW

File: resources/lib/WatchlistFlavor/__init__.py [hunk 1]
```diff
@@ -48,15 +62,20 @@ def login_request(flavor):
             raise Exception("Invalid flavor %s" % flavor)
 
         flavor_class = WatchlistFlavor.__instance_flavor(flavor)
+        login_ts = int(time()) + 2591963
```
Comment (DxCx) [inline resources/lib/WatchlistFlavor/__init__.py]: 2591963??
Lol lets make it a variable please

File: resources/lib/WatchlistFlavor/__init__.py [hunk 2]
```diff
@@ -30,6 +32,20 @@ def get_active_flavor():
 
         return WatchlistFlavor.__SELECTED
 
+    @staticmethod
+    def check_token_expiration():
+        login_ts = control.getSetting(WatchlistFlavor.__LOGIN_TS_KEY)
+        if not login_ts:
+            return True
+
+        expires_in = 2591963 # Seconds until the access_token expires (30 days)
```
Comment (DxCx) [inline resources/lib/WatchlistFlavor/__init__.py]: I meant globally with Capital letters, but this also works, original version was very amature..

Comment (DxCx) [reply resources/lib/WatchlistFlavor/__init__.py]: Want to merge as is or make it global? (Like login keys for example)

Comment (SwagOtaku) [reply resources/lib/WatchlistFlavor/__init__.py]: oh ok, just a sec will make it global

Comment (DxCx) [reply resources/lib/WatchlistFlavor/__init__.py]: Thanks and sorry for the trouble

Comment (SwagOtaku) [reply resources/lib/WatchlistFlavor/__init__.py]: @DxCx  actually just merge as is

Comment (SwagOtaku) [reply resources/lib/WatchlistFlavor/__init__.py]: my bad

**Pull request merged**